fl4health/server/polling.py

```diff
@@ -37,7 +46,17 @@ def _handle_finished_future_after_poll(
 
 
 def poll_client(client: ClientProxy, ins: GetPropertiesIns) -> Tuple[ClientProxy, GetPropertiesRes]:
-    """Get Properties of client"""
+    """
+    Get Properties of client. This is run for each client to extract the properties from the target client
+
+    Args:
+        client (ClientProxy): Client proxy representing one of the clients managed by the server
+        ins (GetPropertiesIns): ins provides any configurations required to help the client retrieve the correct
+            properties
+
+    Returns:
+        Tuple[ClientProxy, GetPropertiesRes]: Returns the resulting properties from the client response.
+    """
```

Comment:
Some of the lines in the docstring do not end with a period. I was wondering if this is on purpose, and if so, why?

Reply:
Not intentional. Just going too fast and missed some punctuation. I'll go through and add them where I find them missing.